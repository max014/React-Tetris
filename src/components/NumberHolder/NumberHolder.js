import React from 'react';
import styles from './NumberHolder.module.css';

export const NumberHolder = (props) => (
	<div className={styles.NumberHolder}>
		<h3>{props.title}</h3>
		<h3>{props.number}</h3>
	</div>
);

export default NumberHolder