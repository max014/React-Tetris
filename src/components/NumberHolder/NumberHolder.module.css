.NumberHolder h3{
	color: white;
}