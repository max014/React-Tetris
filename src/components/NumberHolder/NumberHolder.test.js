import React from 'react';
import { NumberHolder } from './NumberHolder';
import { shallow } from 'enzyme';
import toJson from  'enzyme-to-json';

describe('NumberHolder', () => {
    const props = {title: 'string', number: 0};
    const numberHolder = shallow(<NumberHolder {...props}/>);
    it('renders properly', () => {
        expect(toJson(numberHolder)).toMatchSnapshot();
    })
});