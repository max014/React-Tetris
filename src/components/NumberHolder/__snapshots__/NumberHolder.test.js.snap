// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumberHolder renders properly 1`] = `
<div
  className="NumberHolder"
>
  <h3>
    string
  </h3>
  <h3>
    0
  </h3>
</div>
`;
